great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -542,6 +542,25 @@ def remove_key(self, key):
                 f"Unable to delete object in GX Cloud Store Backend: {repr(e)}"
             )
 
+    def _update(self, key, value, **kwargs):
+        existing = self._get(key)
+        if key[1] is None:
```

Comment:
I think we opt for truthy/falsy in OSS: `if not key[1]`

Good either way but we should be consistent if possible

Reply:
Same note below

Reply:
Gotcha. Will note for future OSS work.